pkg/ctl/cmdutils/nodegroup_flags.go

```diff
@@ -37,7 +37,7 @@ func AddCommonCreateNodeGroupFlags(cmd *cobra.Command, fs *pflag.FlagSet, p *api
 	fs.IntVar(&ng.MaxPodsPerNode, "max-pods-per-node", 0, "maximum number of pods per node (set automatically if unspecified)")
 
 	ng.SSH.Allow = fs.Bool("ssh-access", *ng.SSH.Allow, "control SSH access for nodes")
-	ng.SSH.PublicKeyPath = fs.String("ssh-public-key", *ng.SSH.PublicKeyPath, "SSH public key to use for nodes (import from local path, or use existing EC2 key pair)")
+	ng.SSH.PublicKeyPath = fs.String("ssh-public-key", "", "SSH public key to use for nodes (import from local path, or use existing EC2 key pair)")
```

Comment:
The thing with previous approach that user sees this:
```
      --ssh-public-key string          SSH public key to use for nodes (import from local path, or use existing EC2 key pair) (default "~/.ssh/id_rsa.pub")
```
Now they won't see what's the default, while there is a default.

Reply:
Changed in the line above, what do you think?